pkg/drivers/hyperv/hyperv_windows.go

```diff
@@ -304,6 +305,11 @@ func (d *Driver) Stop() error {
 
 // Remove removes an host
 func (d *Driver) Remove() error {
+	if _, stderr, err := powershell.Execute(`Hyper-V\Get-VM`, d.MachineName); err != nil &&
+		strings.Contains(stderr, "Hyper-V was unable to find a virtual machine") {
+		return nil
+	}
+
```

Comment:
@guillaumerose shouldn't this also for other driver like libvirt/hyperkit when vm is deleted from virtual manager and then try to delete it?
```
$ sudo virsh destroy crc
Domain 'crc' destroyed

$ sudo virsh  list --all
 Id   Name   State
-----------------------
 -    crc    shut off

$ sudo virsh undefine crc
Domain 'crc' has been undefined

$ sudo virsh  list --all
 Id   Name   State
--------------------

$ ./crc delete
Do you want to delete the OpenShift cluster? [y/N]: y
Driver cannot remove machine: Failed to fetch machine 'crc'
```

Reply:
Yes libvirt would be the missing piece. Here the line to change: https://github.com/code-ready/machine-driver-libvirt/blob/master/pkg/libvirt/libvirt.go#L437

If it's ok to do this behaviour change, I will do it 👍

Reply:
It should be good be consistent across all the platform.

Reply:
Fwiw, I'd have a slight preference for having a "VMNotFound" error which we can special case, but noop in this case will also do :)

Reply:
The custom error would force us to add it in machine repo so that we can use it everywhere. A little bit painful :/

Reply:
> The custom error would force us to add it in machine repo so that we can use it everywhere. A little bit painful :/

No worries, as I said, no strong objection to just returning `nil` in this situation :)